src/dfmt/formatter.d

```diff
@@ -747,10 +747,10 @@ private:
                 newline();
             }
         }
-        else if (peekBackIs(tok!"identifier") && (peekBack2Is(tok!"{", true)
-                || peekBack2Is(tok!"}", true) || peekBack2Is(tok!";", true)
-                || peekBack2Is(tok!":", true) || peekBack2Is(tok!",", true))
-                && !(isBlockHeader(1) && !peekIs(tok!"if")))
+        else if (peekBackIs(tok!"identifier")
+                && [tok!"{", tok!"}", tok!";", tok!":", tok!","]
+                .any!((ptrdiff_t token) => peekBack2Is(cast(IdType)token, true))
+                && (!isBlockHeader(1) || peekIs(tok!"if")))
```

Comment:
this is unrelated but I think this is a good change (removes the double negation) so I approve of this

Reply:
Is there anything to do?

Reply:
no I think just waiting for other reviews now if there is anyone who would like to review